### DISCUSSION TITLE
SS Rank seems to be too hard to achieve in mania mode

### DISCUSSION THREAD

**Discussion by Nana-Sakura** [ppy/osu#27903] [Q&A]
I have to mention this again after communicating with some other osu!mania players(Various Rankings).
With osu!lazer, the hit window of Rainbow 300(Perfect) is now variable. 
Most Beatmaps have similar OD, around 8. 
The hit window of Perfect with OD8 is almost the same in both osu!stable and osu!lazer.
Perfect window with osu!stable is 16.5ms.
We can see that if we want to achieve SS, we have to make our judgments in the window of 16.5ms. That is OD 10.58 with osu! in order to make that more clearly. We still have to say that osu!mania is very different from osu!, but we can still find that it is very very very hard for any players. It means that it is almost unachievable for beatmaps with higher SR. Not everybody is likely to be 2ky or other godly accuracy players.
I will have to ask that who can get 1,000,000 on every beatmap. I've already said that everyone has the right to claim SS under reasonable standards. You will know how hard it is to achieve a 1,000,000 on a beatmap, even 0.8*, iff you have ever played osu!mania seriously. Really not that easy, or I have to say, very hard. 
Now in 2024, we can still find 60Hz screens, so how can their users get full score? That's another thing to think seriously.
Great(Previously yellow 300) is now not 100% accuracy, that's OK, but 300 in osu! is absolutely full accuracy, which is told in New Beginnings [tutorial]. I've also said that it can be SS even if it is not 100% accuracy but with all 300. 
At last, I know that boss peppy had started a vote for this issue. But sadly, it is on X(formerly twitter). It is just like collecting surveys at crossroads nearby the house, isn't it. (Many players don't use X.) I do think that voting with osu! clients is much more reasonable. Perfect is Rainbow 300, Great is Yellow 300, Good is 200, OK is 100, Meh is 50 should also be mentioned.

**Comment by MizuIro-H** [comment]
They should use V1 Acc system and V2 Score to make most players satisfied
